go.mod

```diff
@@ -23,14 +23,14 @@ require (
 	github.com/openshift/origin v4.1.0+incompatible
 	github.com/operator-framework/operator-sdk v1.12.0
 	github.com/pkg/errors v0.9.1
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.64.1
```

Comment:
did you run make vendor afterwards, and add the necessary files?

Reply:
yes, no additional changes